README.md

```diff
@@ -5,30 +5,30 @@
 
 # Gargoyle S3Proxy
 
-Template for s3 proxy based on Akka Http.
-
 gargoyle-s3proxy acts as a security layer between s3 user (eg. application using aws sdk) and s3 backend (eg. ceph RadosGW).
-It interacts with
-* [STS Service](https://github.com/kr7ysztof/gargoyle-sts) to:
-    - Validate short-term tokens issued by STS service
-    - Retrieve user information needed to check access policies
-* [Ranger](https://ranger.apache.org/) to get bucket policies. On ranger, custom [ranger plugin](https://github.com/bolkedebruin/rangers3plugin) is used to define policies.
-* S3 Backend (Current setup contains Ceph image with RadosGW)
 
-## Usage
-To run it:
+## How to run
+To test the proxy (both live and integration tests), we need all dependencies to be running. For this we use a `docker-compose.yml` which defines all dependencies, run it using:
 
-* sbt run
+    docker-compose up
 
-## Local Testing
+When all is runnning we can start the proxy:
 
-To test the proxy locally (both live and integration tests), we need all dependencies to be running. For this we use a `docker-compose.yml` which defines all dependencies, run it using:
+    sbt run
 
-    docker-compose up
+## Architecture
 
-### Apache Ranger 
+![alt text](./docs/img/architecture.png)
```

Comment:
I suggest to clearly state that we use two types of tokens (sts token etc.) Also I would replace s3 cmd with s3 request, because it suggests that we only support s3cmd tasks

Reply:
done